custom_components/kamstrup_403/sensor.py

```diff
@@ -216,24 +214,28 @@
         key="97",  # 0x0061
         name="Temp1xm3",
         icon="mdi:thermometer",
-        device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
         entity_registry_enabled_default=False,
     ),
     SensorEntityDescription(
         key="110",  # 0x006E
         name="Temp2xm3",
         icon="mdi:thermometer",
-        device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
         entity_registry_enabled_default=False,
     ),
     SensorEntityDescription(
-        key="113",  # 0x0071
+        key="99",  # 0x0063
         name="Infoevent",
         icon="mdi:eye",
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    SensorEntityDescription(
+        key="113",  # 0x0071
+        name="Infoevent_counter",
+        icon="mdi:eye",
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
```

Comment:
You should also add the serial number, as you manage to find that:
```suggestion
    ),
    SensorEntityDescription(
        key="1001",  # 0x03E9
        name="Serial number",
        icon="mdi:barcode",
        entity_category=EntityCategory.DIAGNOSTIC,
    ),
```